packages/react-relay/modern/ReactRelayGraphQL.macro.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../babel-plugin-relay/BabelPluginRelayMacro');
```

Comment:
This module is named the way it is because `babel-macros` requires the source string to match the regex `/[./]macro(\.js)?$/`, and I import this module in the test suite.

Reply:
You might want to add that as a comment in the file :+1: